etc/config.xml

```diff
@@ -3,7 +3,7 @@
     <default>
         <payment>
             <comfino>
-                <active>1</active>
+                <active>0</active>
```

Comment:
to ma być?

Reply:
Tak, domyślnie powinno być wyłączone jednak.